### DISCUSSION TITLE
Connection error: app.verification-failed-title

### DISCUSSION THREAD

**Discussion by Romed78** [mynaparrot/plugNmeet-server#143] [Q&A]
Hi
we have a problem establishing the connection. At first everything worked but now I get this error message:
app.verification-failed-title
We installed the latest version and Joomla 3.x.

Please help - thanks

**Comment by jibon57** [comment]
Download latest client or change to remote:

<img width="967" alt="image" src="https://user-images.githubusercontent.com/6080277/194591721-65fa34b3-3b85-4763-a66e-d0f7e6f54560.png">
<img width="755" alt="image" src="https://user-images.githubusercontent.com/6080277/194591798-00d3be57-5d6d-48e1-91a9-bbc84ff707ba.png">

**Reply by Romed78** [reply]
this are my seetings

![image](https://user-images.githubusercontent.com/93002016/194594262-86ac5524-3da9-4bf5-b96f-227b33dd6281.png)

![image](https://user-images.githubusercontent.com/93002016/194594379-670c800d-925a-4483-9a46-eb6e71f2e30b.png)

**Reply by Romed78** [reply]
but it doesn't work :-(

**Reply by jibon57** [reply]
OK, you're using demo server. Demo server doesn't guarantee to always working, it may be overloaded or sometime down for testing. I'll encourage you to setup your own server or if you would like managed hosting service then you can contact to @wbobteng : bob[@]mynaparrot.com

**Reply by Romed78** [reply]
This is the erros:

![image](https://user-images.githubusercontent.com/93002016/194594922-401be171-0635-4455-9366-320213770a9f.png)

**Reply by Romed78** [reply]
if we start the meeting from another public IP the meeting works
